onnxruntime/core/providers/rocm/tunable/gemm.cu

```diff
@@ -108,6 +260,22 @@ GEMM(BFloat16, BFloat16) { return CALL_GEMM_WITH_AB_SWAPPED(BFloat16, BFloat16);
 GEMM(double,   float   ) { return CALL_GEMM_WITH_AB_SWAPPED(double,   float   ); }
 GEMM(half,     float   ) { return CALL_GEMM_WITH_AB_SWAPPED(half,     float   ); }
 GEMM(BFloat16, float   ) { return CALL_GEMM_WITH_AB_SWAPPED(BFloat16, float   ); }
+
+BATCHED_GEMM(double,   double  ) { return CALL_BATCHED_GEMM_WITH_AB_SWAPPED(double,   double  ); }
+BATCHED_GEMM(float,    float   ) { return CALL_BATCHED_GEMM_WITH_AB_SWAPPED(float,    float   ); }
+BATCHED_GEMM(half,     half    ) { return CALL_BATCHED_GEMM_WITH_AB_SWAPPED(half,     half    ); }
+BATCHED_GEMM(BFloat16, BFloat16) { return CALL_BATCHED_GEMM_WITH_AB_SWAPPED(BFloat16, BFloat16); }
+BATCHED_GEMM(double,   float   ) { return CALL_BATCHED_GEMM_WITH_AB_SWAPPED(double,   float   ); }
+BATCHED_GEMM(half,     float   ) { return CALL_BATCHED_GEMM_WITH_AB_SWAPPED(half,     float   ); }
+BATCHED_GEMM(BFloat16, float   ) { return CALL_BATCHED_GEMM_WITH_AB_SWAPPED(BFloat16, float   ); }
+
+STRIDED_BATCHED_GEMM(double,   double  ) { return CALL_STRIDED_BATCHED_GEMM_WITH_AB_SWAPPED(double,   double  ); }
+STRIDED_BATCHED_GEMM(float,    float   ) { return CALL_STRIDED_BATCHED_GEMM_WITH_AB_SWAPPED(float,    float   ); }
+STRIDED_BATCHED_GEMM(half,     half    ) { return CALL_STRIDED_BATCHED_GEMM_WITH_AB_SWAPPED(half,     half    ); }
+STRIDED_BATCHED_GEMM(BFloat16, BFloat16) { return CALL_STRIDED_BATCHED_GEMM_WITH_AB_SWAPPED(BFloat16, BFloat16); }
+STRIDED_BATCHED_GEMM(double,   float   ) { return CALL_STRIDED_BATCHED_GEMM_WITH_AB_SWAPPED(double,   float   ); }
+STRIDED_BATCHED_GEMM(half,     float   ) { return CALL_STRIDED_BATCHED_GEMM_WITH_AB_SWAPPED(half,     float   ); }
+STRIDED_BATCHED_GEMM(BFloat16, float   ) { return CALL_STRIDED_BATCHED_GEMM_WITH_AB_SWAPPED(BFloat16, float   ); }
 // clang-format on
 
 #undef CALL_GEMM_WITH_AB_SWAPPED
```

Comment:
Do we need `#undef CALL_BATCHED_GEMM_WITH_AB_SWAPPED` and `#undef CALL_STRIDED_BATCHED_GEMM_WITH_AB_SWAPPED` here?